ansible.properties

```diff
@@ -1,7 +1,7 @@
 category=External Analyzers
 description=Analyze Ansible playbooks.
 homepageUrl=https://github.com/sbaudoin/sonar-ansible
-archivedVersions=2.4.0,2.5.0
```

Comment:
Update:
- I found, in the other plugins, a couple of versions compatible with SQ 9.9 that, I think, should be preserved in the archivedVersions list. I've added comments there.
- Also, I think we should drop the descriptions (and linksà of the versions we completely remove.

Reply:
I dropped the list of archived version lists because the job failed since the referenced versions of SQ weren't available. 
What are you expecting @christophelevis? That we keep version numbers in the `archivedVersions` list, but delete the entire(?) block that describes the version? Delete only the URLs from those blocks?

Reply:
@christophelevis can you take another look, please?